### PR TITLE
add s3s support

### PR DESCRIPTION
**Pull request by mlassnig**

### CODE REVIEW

File: lib/rucio/daemons/conveyor/common.py [hunk 1]
```diff
@@ -334,8 +334,7 @@ def bulk_group_transfer(transfers, policy='rule', group_bulk=200, source_strateg
                       'bring_online': transfer['bring_online'] if transfer['bring_online'] else None,
                       'job_metadata': {'issuer': 'rucio'},  # finaly job_meta will like this. currently job_meta will equal file_meta to include request_id and etc.
                       'overwrite': transfer['overwrite'],
-                      'priority': 3,
-                      's3alternate': True}
```
Comment (mlassnig) [inline lib/rucio/daemons/conveyor/common.py:338]: this had to be removed. newer FTS versions do not need this anymore

File: lib/rucio/core/transfer.py [hunk 2]
```diff
@@ -942,8 +950,7 @@ def get_transfer_requests_and_source_replicas(total_workers=0, worker_number=0,
                     transfers[req_id]['strict_copy'] = strict_copy
                 if use_ipv4:
                     transfers[req_id]['use_ipv4'] = True
-                if archive_timeout and (rses_info[dest_rse_id]['rse_type'] == RSEType.TAPE
-                                        or rses_info[dest_rse_id]['rse_type'] == 'TAPE'):
+                if archive_timeout and (rses_info[dest_rse_id]['rse_type'] == RSEType.TAPE or rses_info[dest_rse_id]['rse_type'] == 'TAPE'):
```
Comment (mlassnig) [inline lib/rucio/core/transfer.py:953]: not part of this patch, but flake8 complained about it

**Review by bari12 — APPROVED** [review]
